### ISSUE TITLE
wrong output

### ISSUE THREAD

**Issue by Wedoria1** [brianhuang1019/Reinforcement-Learning_Chatbot#21]
hey, thanks for the code. I used your pre-trained model. but getting absurd responses. Should I train it to more epochs?
A: Have you heard about 'machine learning and having it deep and structured'?
B: Woods path woods bee sir takes nowhere thompson.
A: Reminds 'a name's working rebels secrets guts procedure fairy missus pain warned procedure ignorant troops wrap famous pain dna wheel troops.
B: Tongue hooked ignorant tongue tongue manner ignorant positively pain warned break real travel sir.
A: Pain las tricks putting ears shack warm ignorant positively arrives brand woods knew jersey domino wrap.
B: Reminds whores pain warned river rio brand.
A: Mob longer. Procedure woody luther wasted tricks specialty assumed window pain scientist.
B: Wigand much shift mud traitor woody stab brand submit submit touches tanks department woody feed pain middle wrap ignorant unlikely.
A: House real policeman term.
B: Mob slaves knee. Meters bat assumed martin woods tongue pit.
A: Jesse electrical reports domino real.
B: Redi whores real. Redi whores real. Desmond real.

**Comment by aflaloe** [comment]
Hi!

I also had absurd responses and tried to run for lots of epochs. The loss seems to decrease indeed but the answers are still non-sense...
@pochih would you have an idea why?

Thanks
